### DISCUSSION TITLE
"Account holder" for Strip - bancontact?

### DISCUSSION THREAD

**Discussion by bmaster001** [pretix/pretix#3837] [Q&A] [answered]
What is this 'account holder' textbox for if the user chooses the Stripe - Bancontact method? I use bancontact almost daily and never has an online shop asked me for this? Is this something Pretix adds, or is it Stripe? 

![image](https://github.com/pretix/pretix/assets/6114149/9d34724d-7f2a-498f-a1b9-3d0c8ccae319)

**Comment by pc-coholic** [comment, marked as answer]
That's indeed a requirement from Stripe - see https://stripe.com/docs/payments/bancontact/accept-a-payment?platform=web&ui=API

Not saying that it makes sense - Stripe is known to collect some obligatory information when it is not really required anymore by the payment method provider...

On the other hand: It might really be required by Bancontact and other shop systems just pass the billing or shipping information...

**Reply by bmaster001** [reply]
Oh, ok, thanks for the quick reply. I guess other systems just take the customer name and send that to Stripe. Is something similar possible with Pretix? We can ask the name and email during step 1 of the checkout process, so technically, this could be sent to Stripe as well?
